	- allow variables to be overridden
	- allow passing TARGET_CFLAGS
	  overriding CCOPTS instead wont work, as this variable is being
	  used to compile host-utils in netem/
	- define LIBUTIL for misc/ss to compile correctly
--- iproute2-2.6.37.orig/Makefile	2011-01-07 18:54:30.000000000 +0100
+++ iproute2-2.6.37/Makefile	2011-01-18 02:19:41.379921947 +0100
@@ -27,10 +27,10 @@ ADDLIB+=dnet_ntop.o dnet_pton.o
 #options for ipx
 ADDLIB+=ipx_ntop.o ipx_pton.o
 
-CC = gcc
-HOSTCC = gcc
-CCOPTS = -D_GNU_SOURCE -O2 -Wstrict-prototypes -Wall
-CFLAGS = $(CCOPTS) -I../include $(DEFINES)
+CC := gcc
+HOSTCC := gcc
+CCOPTS ?= -D_GNU_SOURCE -O2 -Wstrict-prototypes -Wall
+CFLAGS = $(CCOPTS) -I../include $(DEFINES) ${TARGET_CFLAGS}
 YACCFLAGS = -d -t -v
 
 LDLIBS += -L../lib -lnetlink -lutil
@@ -38,6 +38,7 @@ LDLIBS += -L../lib -lnetlink -lutil
 SUBDIRS=lib ip tc misc netem genl
 
 LIBNETLINK=../lib/libnetlink.a ../lib/libutil.a
+LIBUTIL=../lib/libutil.a
 
 all: Config
 	@set -e; \
